dashboard/lib/pd/foorm/rollup_creator.rb

```diff
@@ -0,0 +1,116 @@
+# Create rollup for a set of Foorm responses
+# Rollup will only work if question uses numbers for response options (otherwise question will be ignored)
+# Rollup will include average response for each question, and if question is part of a matrix, average for
+# all responses in the matrix. Currently it only returns matrix averages
```

Comment:
Is only returning matrix averages enough for now, or should it also be returning the other things?

Reply:
matrices is enough for now--the only rollups we show for CSD/CSP are matrices. I will add other types as a follow-up. I think we should support checkbox/radio/rating as well

Reply:
CSF is also currently all matrices